documentation/getting-started.adoc

```diff
@@ -0,0 +1,245 @@
+== Getting started
+
+=== Prerequisites
+
+Kubernetes or OpenShift cluster are required to deploy Strimzi. Strimzi supports all kind of clusters - from public and
+private clouds up to local deployments intended for development purposes. This guide expects that a Kubernetes or
+OpenShift cluster is available and the `kubectl` or `oc` command line tools are installed and configured to connect
+to the running cluster.
+
+When no Kubernetes or OpenShift cluster is available, `Minikube` or `Minishift` can be used. More details can be found
```

Comment:
"When no existing Kubernetes or OpenShift cluster is available, `Minikube` or `Minishift` can be used to create a local cluster." is, perhaps, clearer.